src/main/java/com/revature/ams/Member/MemberService.java

```diff
@@ -1,21 +1,56 @@
 package com.revature.ams.Member;
 
 import com.revature.ams.util.interfaces.Serviceable;
-import com.revature.ams.Member.Member;
 
+import java.util.ArrayList;
+import java.util.List;
+
+// TODO: REVIEW ME
 public class MemberService implements Serviceable<Member> {
+    private List<Member> memberList = new ArrayList<>();
+
     @Override
     public Member[] findAll() {
         return new Member[0];
```

Comment:
Given our member holds onto information like email & password in plain text, we want to prevent the ability for users to simply findAll other users information. But we are still utilizing servicable interface, requiring there to be an overridden findAll method implementation.